service/mcs/mcs.go

```diff
@@ -63,6 +65,7 @@ type Resource struct {
 	Namespace   *string           `json:"namespace,omitempty"`
 	Cost        *float64          `json:"cost,omitempty"`
 	ComputeCost *float64          `json:"computeCost,omitempty"`
+	EbsCost     *float64          `json:"ebsCost,omitempty"`
```

Comment:
```suggestion
	EBSCost     *float64          `json:"ebsCost,omitempty"`
```

[See](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).